Query/dictionary.json

```diff
@@ -0,0 +1,50 @@
+{
+  "caption": "Attribute Dictionary",
+  "description": "The Attribute Dictionary defines schema attributes and includes references to the events and objects in which they are used.",
+  "name": "dictionary",
+  "attributes": {
+    "entity_type_id": {
+      "caption": "Entity Type ID",
+      "description": "The type of entity or object.",
```

Comment:
To follow the OCSF naming conventions, I think we can reuse the `type_id` attribute like Observable does [here](https://github.com/query-ai/ocsf-schema/blob/main/objects/observable.json#L15-L28). Then you'd remove this attribute from this file and move the enum to threat_intelligence.json